Automattic/woocommerce-payments
# Migration of simple requests to request classes

Fixes #6217

#### Changes proposed in this Pull Request



This pull request introduces a new Get_Request class that simplifies the execution of straightforward requests towards the WCPay server. By utilizing this class, existing simple requests are migrated to dedicated request classes. 





#### Testing instructions





* Make sure that all tests are passing
* Make sure that all migrated requests are working as expected.

-------------------


**Post merge**
